templates/full-stack.template.yaml

```diff
@@ -66,7 +66,7 @@ Parameters:
   BuildImageName:
     Description: Docker image for application build
     Type: String
-    Default: aws/codebuild/amazonlinux2-x86_64-standard:2.0
+    Default: aws/codebuild/standard:3.0
```

Comment:
We need a build image with openjdk11 (unless we want to migrate to corretto java at some point)